.github/workflows/build_and_run_chain_simulator_and_execute_system_test.yml

```diff
@@ -115,6 +115,10 @@ jobs:
           latest_commit_hash=$(git rev-parse origin/${current_branch})
           echo "LATEST_COMMIT_HASH=${latest_commit_hash}" >> $GITHUB_ENV
           echo "Latest commit hash: ${latest_commit_hash}"
+          
+          latest_core_go_hash=$(grep 'github.com/multiversx/mx-chain-core-go =>' go.mod | sed -E 's/.*-([a-f0-9]{12})/\1/')
```

Comment:
These actions are somehow wrong.
We shouldn't need to update every possible satellite manually.

Also, These updates are necessary for every satellite project

Reply:
Not my implementation...